### ISSUE TITLE
[review] - average rating NULL or 0 - model vs migration inconsistency

### ISSUE THREAD

**Issue by nakashu** [Sylius/Sylius#4248]
Generated migration for product.average_rating is `DEFAULT NULL` and model and mapping are for NOT NULL.

What should be fixed, migration or model?

Migration:
```
$this->addSql('ALTER TABLE sylius_product ADD average_rating DOUBLE PRECISION DEFAULT NULL');
```

Mapping Product.orm.xml
```
<field name="averageRating" column="average_rating" type="float" />
```

Model Product.php
```
protected $averageRating = 0;
```

**Comment by pjedrzejewski** [comment]
Migration, it should be set to 0 I believe. Thanks for reporting this bug!

**Comment by michalmarcinkowski** [comment]
Fixed in #4254
